### ISSUE TITLE
Distortion Coefficients Enexplained

### ISSUE THREAD

**Issue by cxy1997** [argoai/argoverse-api#81]
The `distortion_coefficients_` in [vehicle_calibration_info.json](https://github.com/argoai/argoverse-api/blob/9edd6f39b9bff297c04129cc1619ddb7c2d16005/tests/test_data/tracking/1/vehicle_calibration_info.json) is a 3-element list. But usually, distortion coefficients [have 4-8 parameters](https://docs.opencv.org/2.4/doc/tutorials/calib3d/camera_calibration/camera_calibration.html). 
Is there an explanation for `distortion_coefficients_`?

**Comment by johnwlambert** [comment]
Hi @cxy1997, we've already undistorted all of the imagery in Argoverse, so you won't need to worry about this. We provided it just in case people were curious about the camera parameters.

OpenCV uses 5 distortion parameters: 3 for radial distortion k1,k2,k3 and 2 for tangential distortion (p1,p2). We set p1,p2 equal to zero, and the 3-element list in `vehicle_calibration_info.json` are radial distortion coefficients k1,k2,k3.
